userProfile/userProfile.mjs

```diff
@@ -0,0 +1,74 @@
+import DatabaseDriver from "../database/driver.mjs"
+
+/**
+Represents a database driver for the TPEN MongoDB database.
+This driver interacts with the private TPEN Mongo Database (users collection).
+*/
+//const database = new DatabaseDriver("mongo")
```

Comment:
Good.  You know you need to do it, but you aren't letting it get in the way of your scope.  This is the correct way to use it.  However, you need to have the correct properties in your `.env` and a local instance of mongo running to get the dbTests to pass.  That is out of your scope for now, but is part of what you need to do next if your next task requires interaction with "the private `tpen` mongodb".